authors/apps/articles/tests/test_article_report.py

```diff
@@ -0,0 +1,73 @@
+from .base import BaseTest
+from rest_framework.views import status
+
+
+class ReportArticleTestCase(BaseTest):
```

Comment:
In base.py there is an author_user and non_author_user defined. Since you've inherited them you can use self.token or self.non_user_token for the token rather than defining another method for that.

Reply:
from base .py I am making use the URL's for login and sign up as well as article_data the user_data so as to get the token as a string.